packages/asc-ui/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -17,10 +17,6 @@ describe('Checkbox', () => {
     ))
   })
 
-  it('should render', () => {
```

Comment:
I am not seeing a replacement test for this one.

Reply:
Well the other tests in this spec already cover that, so I think it's pointless to add one